### DISCUSSION TITLE
Update pyarrow to 11.x or 12.x

### DISCUSSION THREAD

**Discussion by Timost** [datapane/datapane#390] [General]
Hi,
Thank you for datapane 🙏.

Would it be possible to update pyarrow dependency to 11.x or 12.x ? 🙏

Right now, datapane forces us to use 10.0.1 which is missing a few improvements from the later versions.

Thank you for your help.

**Comment by mands** [comment]
Hi @Timost - yes, we have an upcoming release planned for the next few weeks that will bump `pyarrow` up to `12.x`

**Reply by Timost** [reply]
Great news ! Thank you ! 👍

**Comment by mands** [comment]
Fixed in final 0.17.0 release
